README.md

```diff
@@ -10,7 +10,7 @@ GraphQL based data sync server for mobile, with backend integration capabilities
 
 ```
 docker run --rm --name=postgres -p 5432:5432 -e POSTGRES_PASSWORD=mysecretpassword -v `pwd`/examples:/tmp/examples -d postgres
-docker exec postgres psql -U postgres -f /tmp/examples/create_tables.example.sql
+docker exec postgres psql -U postgres -f /tmp/examples/create_example_datasource.sql
```

Comment:
This fails because the table is not created before insert operation. @david-martin

Reply:
Good catch.
I'll update to start the server before creating the example datasource

Reply:
So, what I understand is, Sequlize should create the table. But it didn't do in my case. After I manually created the table and run the server, I saw log statements like "CREATE TABLE etc " in the logs.

Here are the log statements:
```
Executing (default): SELECT "id", "name", "type", "config", "createdAt", "updatedAt" FROM "DataSources" AS "DataSource";
Executing (default): SELECT t.typname enum_name, array_agg(e.enumlabel ORDER BY enumsortorder) enum_value FROM pg_type t JOIN pg_enum e ON t.oid = e.enumtypid JOIN pg_catalog.pg_namespace n ON n.oid = t.typnamespace WHERE n.nspname = 'public' AND t.typname='enum_DataSources_type' GROUP BY 1
Executing (default): CREATE TYPE "public"."enum_DataSources_type" AS ENUM('InMemory', 'Postgres');
Executing (default): SELECT typname, typtype, oid, typarray FROM pg_type WHERE (typtype = 'b' AND typname IN ('hstore', 'geometry', 'geography')) OR (typtype = 'e')
Executing (default): CREATE TABLE IF NOT EXISTS "DataSources" ("id"   SERIAL , "name" VARCHAR(255), "type" "public"."enum_DataSources_type", "config" JSON, "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL, "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL, PRIMARY KEY ("id"));
Executing (default): SELECT i.relname AS name, ix.indisprimary AS primary, ix.indisunique AS unique, ix.indkey AS indkey, array_agg(a.attnum) as column_indexes, array_agg(a.attname) AS column_names, pg_get_indexdef(ix.indexrelid) AS definition FROM pg_class t, pg_class i, pg_index ix, pg_attribute a WHERE t.oid = ix.indrelid AND i.oid = ix.indexrelid AND a.attrelid = t.oid AND t.relkind = 'r' and t.relname = 'DataSources' GROUP BY i.relname, ix.indexrelid, ix.indisprimary, ix.indisunique, ix.indkey ORDER BY i.relname;
```
It first does a "SELECT" and in later stages it does "CREATE TABLE IF NOT EXISTS".

Reply:
Other than this, progress so far looks good.

Reply:
@aliok I've updated the README steps, and changed the order on startup to fully initialise models first (i.e. create if not exists) before trying to list any.

i.e. this https://github.com/aerogear/data-sync-server/pull/9/commits/8f83102c13e4bafd78687e39b1a33f16e760c2ed